aop/src/main/kotlin/io/micronaut/aop/util/KotlinInterceptedMethodHelper.kt

```diff
@@ -35,8 +35,8 @@ internal object KotlinInterceptedMethodHelper {
     suspend fun handleResult(result: CompletionStage<*>, isUnitValueType: Boolean): Any? = suspendCoroutine { continuation ->
         result.whenComplete { value: Any?, throwable: Throwable? ->
             if (throwable == null) {
-                val result = Result.success(value ?: if (isUnitValueType) Unit else null)
-                continuation.resumeWith(result)
+                val res = Result.success(value ?: if (isUnitValueType) Unit else null)
```

Comment:
This change seems a bit superfluous is there a reason for renaming the symbol from 
result to res? I mean you could make a case for a more explicit name but in this case 
res does not feel like an improvement over result.

Reply:
@chalkedgoose The problem is that you use two variables with the same name - a method parameter and a local variable in the lambda. This makes your code less clear, because it's not clear. It's not obvious at what point you're using the parameter, but at what variable. She told me it was an idea inspection, I didn't see it myself. Although, it's strange why sonar didn't catch the smell code before I did

Reply:
I think it is a good change. Having different variable names is more clear.